js/client-teambuilder.js

```diff
@@ -109,6 +109,7 @@
 		update: function () {
 			teams = Storage.teams;
 			if (this.curTeam) {
+				this.ignoreEVLimits = (this.curTeam.gen < 3) || (this.curTeam.format === 'classichackmons');
```

Comment:
Well Custom Game isn't a ladder so you can't select it as your format in the teambuilder (unless you do it in import/export). To answer the question, I believe Custom Game does ignore EV limits.